mtest/libmscore/compat206/lidemptytext-ref.mscx

```diff
@@ -241,6 +241,7 @@
         <barNoteDistance>1.2</barNoteDistance>
         <pedalPosBelow x="0" y="0"/>
         <trillPosAbove x="0" y="0"/>
+        <createMultiMeasureRests>1</createMultiMeasureRests>
```

Comment:
Why is this needed now? I don't see anything in the rest of this PR that might explain this

Reply:
It isn't needed anmore it seems...